sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -899,6 +899,39 @@ export interface PhoneNumberIdentifierModel {
     value: string;
 }
 
+// @public
+export interface PolicyInsightsPolicyStateChangedEventData {
```

Comment:
the three of these look identical... can we use a type alias instead of re-declaring them?